### PR TITLE
update to split on training in one year, test after a date

### LINKED ISSUE #390: Train on 2020.  Test on 2021.  No validation set?
To align with ESO's PV forecasts, let's configure `nowcasting_dataset` to use 2020 for the training set, and use 2021-01-01 to 2021-08-31 for the test set.

Do we need a separate validation set?  I'm not sure we do, given that we're unlikely to use early stopping.  And, "just" a year of training data feels a bit tight, so I'm keen to make sure we use as much of our data as possible for training.  What do you guys think?

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

change split method to
values were hard in by a issue has been made to mvoe this to a config file - https://github.com/openclimatefix/nowcasting_dataset/issues/426

Fixes #390 

## How Has This Been Tested?

unittests

- [ ] No
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by JackKelly** [comment]
I think hard coding is fine for now :slightly_smiling_face: 

But please create a new issue to remind us to make this into a config option in early 2022 :slightly_smiling_face:

**Review by JackKelly — APPROVED** [review]
LGTM! Thanks!  Didn't realise it would be this simple:  Your splitting code is nice and easy to use :slightly_smiling_face:

**Comment by peterdudfield** [comment]
> LGTM! Thanks! Didn't realise it would be this simple: Your splitting code is nice and easy to use 🙂

Thats the beauty of making everything module
